src/bots/Bot.js

```diff
@@ -245,6 +245,19 @@ export default class Bot {
   }
 
   getSSEDisplayError(event) {
+    if (
+      event?.source?.xhr?.status === 403 &&
+      this.constructor._isUseCloudflareTurnstile
+    ) {
+      return `${i18n.global.t(
+        "error.solveCAPTCHA",
+      )}\n${this.getBotLoginHyperlink()}`;
+    }
     return `${event?.source?.xhr?.status}\n${event?.source?.xhr?.response}`;
   }
+
+  getBotLoginHyperlink() {
```

Comment:
```suggestion
  getLoginHyperlink() {
```

The function belongs to a bot. No need to mention `bot` again.